src/NewTools-Debugger/StDebuggerConfigurationCommandTreeBuilder.class.st

```diff
@@ -1,3 +1,11 @@
+"
+I build the default command tree for the debugger configuration.
+The command classes I instantiate are specified under the ""accessing - commands classes"" protocol.
+The group names and descriptions are specified under the ""accessing"" protocol.
+
+You can evolve the default groups names and descriptions and the default command classes, but you cannot extend me.
```

Comment:
what do you mean with this sentence ?

Reply:
Poor phrasing, I meant: the commands statically configured in the class can be modified there, but by default the debugger command tree builders are not meant to be specialized through subclassing.
I will update the comment.